sdk/storage/Azure.Storage.Blobs/tests/AppendBlobClientTests.cs

```diff
@@ -606,6 +606,105 @@ public async Task AppendBlockAsync_EncryptionScope()
             Assert.AreEqual(TestConfigDefault.EncryptionScope, response.Value.EncryptionScope);
         }
 
+        [RecordedTest]
+        [Ignore("https://github.com/Azure/azure-sdk-for-net/issues/20501")]
```

Comment:
can this be playbackonly?

Reply:
Nope, because I am not able to record this test.  Identity SAS only works in canary and prod, and this feature and the 2020-10-02 service version are not currently enabled in canary.